pyproject.toml

```
@@ -41,3 +41,32 @@ python-source = "python"
 python-packages = ["fortitude"]
 manifest-path = "fortitude/Cargo.toml"
 strip = true
+
+[tool.rooster]
+major_labels = []  # Ruff never uses the major version number
```

Tool agnostic comments:
```suggestion
major_labels = []  # Major version number never used
```